public/electron/clients/billing/endpoints/account.js

```diff
@@ -0,0 +1,50 @@
+const DEFAULT_PATH = '/account';
+
+/**
+ * @typedef {Object} CurrentUsageInfo
+ * @property {number} storage
+ * @property {number} bandwidth
+ */
+
+/**
+ * @this {import('../client.js')}
+ * @param {Object} payload
+ * @param {string} payload.token
+ * @returns {import('axios').AxiosResponse<CurrentUsageInfo>}
+ */
+function getCurrent(payload) {
```

Comment:
`getCurrentUsageInfo` could be a better naming

Reply:
fixed 1b83e2e4658207e0880c743e1d3a51d64249ab35